test/event_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/event'
+
+class EventTest < Minitest::Test
+  def setup
+    @event = Event.new("Curling", [24, 30, 18, 20, 41])
+  end
+
+  def test_that_event_exists
+    assert_instance_of Event, @event
+  end
+
+  def test_does_event_have_a_name?
+    assert_equal "Curling", @event.name
+  end
+
+  def test_that_event_stores_ages
+    assert_equal [24, 30, 18, 20, 41], @event.ages
+  end
+
+  def test_does_event_have_an_age_range?
+    assert true, @event.ages.class == Array
+  end
+
+  def test_array_doesnt_include_Strings
```

Comment:
Same comment as above. This first test already shows us that there are no Strings in the Array